python/lsst/ts/mtdome/schema/command.py

```diff
@@ -842,6 +844,55 @@
           }
         }
       }
+    },
+    {
+      "if": {
+        "properties": {
+          "command": {
+            "const": "resetDrivesAz"
+          }
+        }
+      },
+      "then": {
+        "properties": {
+          "parameters": {
+            "type": "object",
+            "properties": {
+              "reset": {
+                "type": "array",
+                "minItems": 5,
+                "maxItems": 5,
+                "items": [
+                  {
+                    "type": "integer"
+                  }
+                ]
+              }
+            },
+            "required": [
+              "reset"
+            ],
+            "additionalProperties": false
+          }
+        }
+      }
+    },
+    {
+      "if": {
+        "properties": {
+          "command": {
+            "const": "calibrateAz"
+          }
+        }
+      },
+      "then": {
+        "properties": {
+          "parameters": {
+            "type": "object",
+            "additionalProperties": false
+          }
+        }
+      }
```

Comment:
For future work I think you could dramatically simplify this code and maintenance as follows:
* Store the properties for each command as a dict of command_name: yaml_snippet that describes *only* the command parameters. For example (using yaml instead of json, because it's shorter, but suit yourself):
```
command_parameters = dict(
    moveAz="""
        position:
          type: number
        velocity:
          type: number
        """,
    moveEl="""
        position:
          type: number
        """,
    # ...
)

def make_schema_from_parameters(params_yaml):
    full_yaml = f"""
$schema: "http://json-schema.org/draft-07/schema#"
type: object
properties:
{params_yaml}
additionalProperties: false
"""
    return yaml.safe_load(full_yaml)

command_schema = {
    cmd: make_schema_from_parameters(params)
    for cmd, params in command_parameters.items()
}
```
Note that the yaml for each command is far more indented than it needs to be, and has some extra newlines, but yaml is OK with that. The `textwrap` built-in library module can easily change the indentation of a triple-quoted string, but it doesn't seem to be needed in this case.

I ran the above with the following snippet:
```
for cmd, schema in command_schema.items():
    print(f"{cmd}: {schema}")
```
and got:
```
moveAz: {'$schema': 'http://json-schema.org/draft-07/schema#', 'type': 'object', 'properties': {'position': {'type': 'number'}, 'velocity': {'type': 'number'}}, 'additionalProperties': False}
moveEl: {'$schema': 'http://json-schema.org/draft-07/schema#', 'type': 'object', 'properties': {'position': {'type': 'number'}}, 'additionalProperties': False}
```

Reply:
A great suggestion and I will consider that for a future ticket.